src/main/java/com/internetitem/logback/elasticsearch/AbstractElasticsearchPublisher.java

```diff
@@ -134,13 +150,6 @@ public void run() {
 							// all done
 							working = false;
 							return;
-						} else {
```

Comment:
This was all bad here. Basically we only ejected and stopped sending the same request over and over again, if we stopped getting new logs. _eventsCopy_ holds the new events (logs) since last execution.